src/common/Statistics.cpp

```diff
@@ -391,6 +397,53 @@ void Statistics::print()
             , totalTimeGettingSoIPhasePatternMicro
             , printPercents( totalTimeGettingSoIPhasePatternMicro,
                              timeMainLoopMicro ) );
+
+    printf( "\t--- Context dependent statistics ---\n" );
```

Comment:
We printed NUM_POPS and NUM_SPLITS above. How are NUM_CONTEXT_PUSHES/POPS different from those? (Do we want to print these statistics together?)

Reply:
The semantics actually differ, since push and pop above are the old SmtCore implementation. I printed them here, since these are actual context operations and they can be used to gauge the average cost of a push/pop operation. In the new SmtCore the two semantics will actually match and I would still want the statistics shown in both places. Maybe even introduce average calculation or something.